package.json

```diff
@@ -45,6 +45,8 @@
     "clipboardy": "1.1.4",
     "debounce": "1.0.2",
     "get-port": "3.1.0",
+    "ignore-by-default": "1.0.1",
```

Comment:
I guess we can just copy the array it returns, to not add the extra dep 👍

Reply:
I'm agree. The reason I used the dependency is because is more easy add common files to ignore around projects 😄